packages/gateway/src/search/index.ts

```diff
@@ -28,9 +18,5 @@ export async function getRecordById(
     }
   })
 
-  if (!res.ok) {
-    throw new Error(`Failed to fetch record ${recordId}`)
-  }
-
```

Comment:
Isn't it better to keep this?

Reply:
I removed the whole file as there were now references to it from anywhere anymore

Reply:
ah okay